src/hal/ticks.h

```diff
@@ -8,7 +8,10 @@
 #include "util/types.h"
 
 struct ticks_hal_t {
+	/* 'nop' for x milisecs */
 	void (*delay_ms)(u32 ticks);
+	/* 'nop' for x microsecs */
```

Comment:
Maybe this would be better as "wait for x ...". The nop detail seems implementation specific.

Reply:
you're right.